openshift-4.0/group.yml

```diff
@@ -155,6 +155,11 @@ sources:
     branch:
       target: release-{MAJOR}.{MINOR}
       fallback: master
+  ignition-rpm:
+    url: https://src.fedoraproject.org/rpms/ignition.git
```

Comment:
Using this directly from fedora is bound to be problematic because of the sheer between the build deps/env. I can see at least this in the spec file will be a problem:

```
# e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 aarch64 %{arm}}
```

We'll have to rewrite that somehow for the four arches we do support. Unless go_arches magically evaluates to what we want there, which seems dubious to me.

```
# If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}
```
This will work as long as golang 1.9 is OK, which is not an invariant I would count on in Fedora. We'd need to specify `golang >= 1.10`

I think we need to take a long look and have upstream discussion on this :) Also this is probably related to RHCOS - what they use in their boot process.